concent_api/common/decorators.py

```diff
@@ -169,9 +171,45 @@ def wrapper(request, client_message, client_public_key, *args, **kwargs):
             try:
                 if database_name is not None:
                     sid = transaction.savepoint(using=database_name)
-                response_from_view = view(request, client_message, client_public_key, *args, **kwargs)
+                try:
+                    response_from_view = view(request, client_message, client_public_key, *args, **kwargs)
+                except DatabaseError as e:
+                    # This section is to catch Database errors caused by multiple requests about one Subtask.
+                    # This requests are waiting and retrying.
+                    log_string_message(
```

Comment:
You are logging basically the same message twice -here and few lines below...

Reply:
They are very similar, but this 'first' is after first handling error. Each other is after another catch and retry so i think it is ok. They are not duplicated  and each one is important.